docs/create/heroku/herokuruby/index-herokuruby.mdx

```diff
@@ -0,0 +1,201 @@
+---
+id: index-herokuruby
+title: Deploy a Ruby app on Heroku using Redis
+sidebar_label: How to deploy a Ruby based application on Heroku using Redis 
+slug: /create/heroku/herokuruby
+authors: [ajeet]
+---
+
+import RedisCard from '@site/src/theme/RedisCard';
+
+Heroku is a popular PaaS offering that allows software developers to easily deploy their code without worrying about the underlying infrastructure. By using a simple 'git push heroku' command, developers are able to deploy their application flawlessly. This platform offers support for a wide range of programming languages such as Java, Ruby, PHP, Node.js, Python, Scala, and Clojure. 
+
+Here's a quickstart guide to deploy Ruby apps over Heroku using Redis. We will be deploying a sample Leaderboard app and will be using company valuation and stock tickers as its domain.
+
+
+### Step 1. Create Redis Enterprise Cloud
+
+Create your free Redis Enterprise Cloud account by visiting [this link](https://redis.com/try-free)
+
+![recloud](try-free.png)
+
+[Follow this link to create a Redis Enterprise Cloud](/create/rediscloud) subscription and database as shown below:
+
+
+![heroku](launch_database.png)
+
+The database endpoint URL is unique for all and hence might be different in your case. Save it for future reference.
+
+
+
+### Step 2. Create a Heroku account
+
+If you are using Heroku for the first time, create your new Heroku account [through this link](https://signup.heroku.com/login)
+
+![heroku](create_heroku.png)
+
+
+### Step 3. Install Heroku CLI on your system
+
+   ```macos
+    brew install heroku
+   ```
+
+### Step 4. Login to Heroku
+
+   ```bash
+    heroku login
+    heroku: Press any key to open up the browser to login or q to exit:
+    Opening browser to https://cli-auth.heroku.com/auth/cli/browser/XXXXXXXXXXA
+    Logging in... done
+    Logged in as your_email_address
+   ```
+
+### Step 5. Connect your application to Redis Enterprise Cloud
+
+For this demonstration, we will be using a [Sample Redis Leaderboard app](https://github.com/redis-developer/basic-redis-leaderboard-demo-ruby)
+
+#### Clone the repository
+
+   ```bash
+    git clone https://github.com/redis-developer/basic-redis-leaderboard-demo-ruby
+   ```
+
+
+
+Run the commands below to get a functioning Git repository that contains a simple application as well as a package.json file.
+
+
+```
+heroku create
+Creating app... done, ⬢ thawing-shore-07338
+https://thawing-shore-07338.herokuapp.com/ | https://git.heroku.com/thawing-shore-07338.git
+```
+
+
+
+### Step 6. Setting up Environmental variable
+
+Go to the Heroku dashboard, click "Settings" and set REDIS_HOST and REDIS_PASSWORD under the Config Vars.
```

Comment:
Just checking, do you also need to set the Redis port?

Reply:
No. REDIS_HOST and REDIS_PASSWORD is sufficient.